pages/api/login.js

```diff
@@ -0,0 +1,114 @@
+import bcrypt from 'bcryptjs';
```

Comment:
can you use the nextauth library for authentication instead? like we had in the prisma tutorial:
https://vercel.com/guides/nextjs-prisma-postgres

similar code for reference:
https://github.com/Tek58/my_scheduler/blob/Create-authentication/pages/api/auth/%5B...nextauth%5D.ts